core/server/common/src/main/java/alluxio/web/WebServer.java

```diff
@@ -45,6 +45,7 @@
 public abstract class WebServer {
   private static final Logger LOG = LoggerFactory.getLogger(WebServer.class);
   private static final String DISABLED_METHODS = "TRACE,OPTIONS";
+  private static final String THREAD_DUMP_PATH = "/api/v1/common/thread_dump";
```

Comment:
Why you changed the `stack` path which is user interface level, suggest to keep backward compatible, in another world, it should keep consistent with `routePaths.tsx`

Reply:
This URL is reused by the both frontend and backend, and you can find the unexcepted behavior by refreshing page in the stacks page. Also I think api interface should always start with `/api` to ensure consistency. So i change the path.

Reply:
@tieujason330  is this thread dump path follows convention?

Reply:
Please make this configurable by a property, as we are hearing different opinions about the path.

Reply:
The frontend needs to use this api and if it is configurable then it means that the frontend needs to additionally fetch the configuration and thus increase the complexity. I don't think it's necessary to make this configurable.

Reply:
i think this is fine since it is consistent with `/api/v1/master` and `/api/v1/worker`

Reply:
If we change the `v1` to `v2` in the webui code (probably the jetty part?) this will be an incompatibility. @tieujason330 Do you think there's a way to at least leave a marker so we will notice before upgrade?

Reply:
@jiacheliu3 Is it ok if I modify the code in this way?  I think this will help us to notice.
```suggestion
  private static final String THREAD_DUMP_PATH = REST_API_PREFIX + "/common/thread_dump";
```

Reply:
can use constant `REST_API_PREFIX` in `Constants.java` for consistency. there isn't an automated way to track version changes between server and ui code. we'll manually have to keep track